models/addressable_test.go

```diff
@@ -36,16 +36,13 @@ var TestAddressable = Addressable{Timestamps: testTimestamps, Name: testAddrName
 var EmptyAddressable = Addressable{}
 
 func TestAddressable_MarshalJSON(t *testing.T) {
-	var resultTestBytes = []byte(TestAddressable.String())
-	var resultEmptyBytes = []byte(EmptyAddressable.String())
 	tests := []struct {
 		name    string
 		a       Addressable
 		want    []byte
 		wantErr bool
 	}{
-		{"successful marshal", TestAddressable, resultTestBytes, false},
-		{"successful empty marshal", EmptyAddressable, resultEmptyBytes, false},
+		{"successful empty marshal", EmptyAddressable, []byte(TestEmptyJSON), false},
```

Comment:
Missed this on first review.  Pull TestEmptyJSON (which is now being used in multiple model tests ) into own file.

Reply:
Do you want all files to have only local test variables, or do you want to limit this scope to this constant?

Reply:
Preference is to not have one test take a dependency on constants or variables defined in a separate test (but within the same package).  Extract the commonly used dependencies defined in a single test file into their own file/module.

Reply:
> Preference is to not have one test take a dependency on constants or variables defined in a separate test (but within the same package). Extract the commonly used dependencies defined in a single test file into their own file/module.

You moved constants from test file that are only currently being used by the test.  The ask was to move those that were being used across tests, not to move all constants into a common file/module.

I think I'm missing something...